packages/checkout-ui-extensions/src/components/Icon/Icon.ts

```diff
@@ -22,14 +22,19 @@ export type IconSource =
   | 'disabled'
   | 'errorFill'
   | 'geolocation'
+  | 'grid'
+  | 'hamburger'
   | 'info'
+  | 'infoFill'
   | 'list'
   | 'locateMe'
   | 'lock'
   | 'map'
   | 'marker'
   | 'minus'
   | 'mobile'
+  | 'note'
+  | 'verticalDots'
```

Comment:
This type change would be an enhancement, not a breaking change as it is additive in nature.